sdk/storage/azblob/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.1.0-beta.2 (Unreleased)
 
 ### Features Added
+* Added `AccessTierCold`.
```

Comment:
I would word it as 'Added support for Cold tier'

Reply:
Done!